yarn.lock

```diff
@@ -0,0 +1,12342 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We don't use Yarn so I don't understand what this is doing here. This file should probably be deleted.